index.js

```diff
@@ -27,10 +28,20 @@ function fastifyView (fastify, opts, next) {
   const includeViewExtension = opts.includeViewExtension || false
   const prod = typeof opts.production === 'boolean' ? opts.production : process.env.NODE_ENV === 'production'
   const defaultCtx = opts.defaultContext || {}
+  const layoutFileName = opts.layout
+
+  if (layoutFileName) {
+    try {
+      readFileSync(join(templatesDir, getPage(layoutFileName, 'hbs')))
```

Comment:
Is this just a check on access?

Reply:
Yes. If an user passes a layout but the layout is not readable, this would return an error to them.

Reply:
Would you mind using https://nodejs.org/api/fs.html#fs_fs_accesssync_path_mode instead?

Reply:
No problem.